src/components/common/header/NavBar.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '../../../../public/js/NavBar';
+import logo from '../../../../public/images/logo.png';
+import business from '../../../../public/images/business.svg';
+import fitness from '../../../../public/images/fitness.svg';
+import food from '../../../../public/images/food.svg';
+import gaming from '../../../../public/images/gaming.svg';
+import login from '../../../../public/images/login.png';
+import programming from '../../../../public/images/programming.svg';
+import started from '../../../../public/images/started.png';
+import travel from '../../../../public/images/travel.svg';
+import more from '../../../../public/images/more.svg';
+import menu from '../../../../public/images/menu.svg';
+
+const NavBar = () => (
```

Comment:
I think the implementation of this should be greatly improved. It has too many lines of code that should be separated into different components, it also doesn't account for logged in users.

The `nav` element should be contained in its own component and should contain only navigation links, this should be populated with categories fetched from the backend via a for loop or map function, so you do not have to write categories out one by one like this. So please create mock data that mimics the exact response from the database when you fetch categories and provide it as props to the nav component, refer to [this](https://github.com/andela/valinor-ah-frontend/blob/46f7f7f4ce2e178e2d25b34a2e81f8efbbfd3293/mockdata/articles.js) and its usage [here](https://github.com/andela/valinor-ah-frontend/blob/46f7f7f4ce2e178e2d25b34a2e81f8efbbfd3293/src/components/home/HomePage.js#L50). 

This should be separate from the component that houses the login and sign up buttons, as this component should also account for things like the users profile image, add new post icon, new notification icon, the drop-down user menu and other things that should be displayed when a user is logged in as shown in the mock ups below.
<img width="265" alt="screen shot 2018-12-08 at 11 22 51 am" src="https://user-images.githubusercontent.com/30160407/49685043-cafb0e00-fadc-11e8-8532-2aa7da5163a1.png">. 

All these changes will make the code neater, easier to read and easy for whoever is going to use it as they would not have to start restructuring your code.

Reply:
@KvNGCzA can you check my implementation now?

Reply:
checking....